app/models/alerts.py

```diff
@@ -30,7 +30,11 @@ def last_updated_date(self):
         return AlertDate(self.last_updated)
 
     @classmethod
-    def from_yaml(cls, path):
+    def load(cls):
```

Comment:
It was unclear to me reading through this why this method exists when it solely calls another method. I can also see a mix of `Alerts.load()` and `Alerts.from_yaml()` in the code - I'm not sure how you've decided which one to use in each case. I understand this will change in future to have a different function, but it might be helpful to add a comment in the meantime.

Reply:
Sure, I've amended https://github.com/alphagov/notifications-govuk-alerts/pull/163/commits/354de0209831c5e242d90fa67ad1f76ca13b7862 to add a comment here 👍.